config/environments/local/local.genesis.config.json

```
@@ -1,5 +1,12 @@
 {
+  "L1Config" : {
+		"L1ChainID": 1337,
+		"PoEAddr": "0x610178dA211FEF7D417bC0e6FeD39F05609AD788",
+		"MaticAddr": "0x5FbDB2315678afecb367f032d93F642f64180aa3",
+		"GlobalExitRootManagerAddr": "0x2279B7A0a67DB372996a5FaB50D91eAA73d2eBe6"
```

I think this is a good oportunity to change this names to:
```suggestion
		"L1ChainID": 1337,
		"PolygonZkEVMAddress": "0x610178dA211FEF7D417bC0e6FeD39F05609AD788",
		"MaticAddress": "0x5FbDB2315678afecb367f032d93F642f64180aa3",
		"PolygonZkEVMGlobalExitRootAddress": "0x2279B7A0a67DB372996a5FaB50D91eAA73d2eBe6"
```
I would be good to ask the protocol team to change the format of the json to include this parameters automatically instead having to create it manually